rlwe/params.go

```diff
@@ -357,7 +372,11 @@ func (p Parameters) CopyNew() Parameters {
 	qi, pi := p.qi, p.pi
 	p.qi, p.pi = make([]uint64, len(p.qi)), make([]uint64, len(p.pi))
 	copy(p.qi, qi)
-	copy(p.pi, pi)
+	p.ringQ, _ = ring.NewRingFromType(1<<p.logN, p.qi, p.ringType)
```

Comment:
I think we should remove this method since parameters are now immutable and read only

Reply:
Agreed.

Reply:
For now, I only deprecated the method:
- to not make the code that might use it crash
- because `Parameter` still has one mutable method (`UnmarshalBinary`)